addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/config/RFXComDeviceConfiguration.java

```diff
@@ -15,11 +15,13 @@
  */
 
 public class RFXComDeviceConfiguration {
+    public static final String PULSE_LABEL = "pulse";
```

Comment:
@Jamstah it feels like it should be part of the RFXComDeviceConfiguration but if you agree I can move all the device configuration fields here :-)